apps/dashboard/app/views/files/_spinner.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="text-center text-primary d-none p-500" id="tloading_spinner">
```

Comment:
What is the `p-500` class here? I can't seem to find a definition for it./

Reply:
Whoops I was missing around with the styling and forgot to delete that.